### ISSUE TITLE
This project is insanely amazing !!!

### ISSUE THREAD

**Issue by bhupesh-sf** [av/harbor#17]
Hey,

Thanks for putting this up. This is something I face regularly. Every time, there is a tool I want to try I have to take the pain of all the setup and issues faced during that. 

I always wondered, if there could be a unified solution that can help to simply plug and play any tool and this is what I got here.

I hope we will get the apple silicon support soon and be able to try out the stuff which I still cannot.

**Comment by lee-b** [comment]
Agree, I had a similar set up with lots of docker compose files, but this is very nicely polished and integrated.  I'll be using it in future, and contributing where I can.

**Comment by av** [comment]
Thanks for the feedback. On the Apple Silicon point, Harbor now has `mlx` and `omlx` backends. They run inference natively on macOS for Metal access and expose it to Harbor services through a proxy. The setup is documented here: https://github.com/av/harbor/blob/main/docs/8.8-Run-MLX-on-Apple-Silicon-with-Harbor.md. I verified both handles are listed by `./harbor.sh ls` on main.
